valgrind/src/data.rs

```diff
@@ -152,32 +152,13 @@ pub struct Proposal {
 pub struct AdvisorReview {
```

Comment:
Is there any reason why do we have extra types here instead of using the ones from catalyst-toolbox or vit-servicing-
station directly?

Reply:
It was done by mistake initially but now we see it is good that we have it from server data. then we can catch all discrepancies between client and server at first possible moment.